sql1.txt

```diff
@@ -6,24 +6,28 @@ SELECT * FROM Products LIMIT 5;
 1. Use an SQL SELECT statement to retrieve the first 10 rows of the Customer table, ordered by CustomerName.
 Paste your SQL statement below:
 
+SELECT * FROM Сustomers ORDER BY СustomerName LIMIT 10;
 
 2. Use an SQL Select statement to retrieve the names and prices for all products that cost less than $20.
 Paste your SQL statement below:
 
+SELECT ProductName, Price FROM Products WHERE price < 20;
 
 3. Retrieve all employees whose last name starts with C.  Paste your SQL statement below:
 
+SELECT lastname FROM employees WHERE lastname LIKE "C%";
 
 4. Retrieve the order ID and customer name for all orders where the customer name starts with A.
 Here you will have to do a join of the Orders table and the Customers table.  Paste your SQL
 statement below.
 
+SELECT Orders.OrderID, Customers.CustomerName FROM Orders JOIN Customers ON Orders.CustomerID = Customers.CustomerID WHERE CustomerName LIKE 'A%';
 
 5. Retrieve the list of customers, ordered by customer name, along with their order IDs.  Note
 that not every customer has an order, but be sure that you include those customers without orders
 in the results of the query.  Paste your SQL statement below.
 
-
+SELECT * FROM Customers LEFT JOIN Orders ON  Customers.CustomerID = Orders.CustomerID ORDER BY CustomerName;
```

Comment:
The wording of the question is a bit confusing, but by "Retrieve the list of customers... along with their order IDs", it seems they mean to retrieve only the customer names and their order IDs.

So, you just need to change `SELECT *` to `SELECT Customers.CustomerName, Orders.OrderID`.

Reply:
Done